assets/js/blocks/cart-checkout/cart/test/block.js

```diff
@@ -13,16 +13,19 @@ import { defaultCartState } from '../../../../data/default-states';
 
 describe( 'Testing cart', () => {
 	beforeEach( async () => {
-		fetchMock.mockResponseOnce( ( req ) => {
-			if ( req.url.match( /wc\/store\/cart/ ) ) {
-				return Promise.resolve( JSON.stringify( previewCart ) );
-			}
-		} );
```

Comment:
this was moved from `beforeEach` to the `'renders cart if there are items in the cart'` test because it caused an issue with the empty cart test, as it triggered to calls, causing `toHaveBeenCalledTimes` to always have two (one for the inline in that test, and one for this).